src/main/scala/com/cognite/sdk/scala/common/login.scala

```diff
@@ -12,14 +13,39 @@ class Login[F[_]](val requestSession: RequestSession[F]) {
   implicit val loginStatusDecoder = deriveDecoder[LoginStatus]
   implicit val dataLoginStatusDecoder = deriveDecoder[DataLoginStatus]
   def status(): F[LoginStatus] =
-    requestSession
-      .sendCdf { request =>
-        request
-          .get(uri"${requestSession.baseUri}/login/status")
-          .response(asJson[DataLoginStatus])
-          .mapResponse {
-            case Left(value) => throw value.error
-            case Right(value) => value.data
-          }
-      }
+    new UrlValidator().isValid(requestSession.baseUri.toString()) match {
```

Comment:
Can we use sttp's `Uri` class to make this check, instead of adding another dependency?

Reply:
`URI` doesn't sound like a bad option, if sttp's `Uri.parse` doesn't do anything useful.